addGrowthRate_function.R

```diff
@@ -0,0 +1,44 @@
+require(lubridate)
```

Comment:
Well, that was the recommendation from Ms Hillary -- who I copied as I went through my first package creation flow.

http://hilaryparker.com/2014/04/29/writing-an-r-package-from-scratch/